backend/dataall/modules/omics/cdk/env_role_omics_policy.py

```diff
@@ -0,0 +1,43 @@
+from aws_cdk import aws_iam as iam
+
+from dataall.core.environment.cdk.env_role_core_policies.service_policy import ServicePolicy
+from dataall.modules.omics.services.omics_permissions import CREATE_OMICS_RUN
+
+
+class OmicsPolicy(ServicePolicy):
+    """
+    Creates an Omics policy for accessing and interacting with Omics Projects
+    """
+
+    def get_statements(self, group_permissions, **kwargs):
+        if CREATE_OMICS_RUN not in group_permissions:
+            return []
+
+        return [
+            iam.PolicyStatement(
+                sid='OmicsWorkflowActions',
+                actions=['omics:ListWorkflows', 'omics:GetWorkflow', 'omics:StartRun'],
+                resources=[
+                    f'arn:aws:omics:{self.region}:{self.account}:workflow/*',
+                    f'arn:aws:omics:{self.region}::workflow/*',
+                ],
+            ),
+            iam.PolicyStatement(
+                sid='OmicsRunActions',
+                actions=['omics:ListRuns', 'omics:DeleteRun', 'omics:GetRun', 'omics:ListRunTasks', 'omics:CancelRun'],
+                resources=[
+                    f'arn:aws:omics:{self.region}:{self.account}:run/{self.resource_prefix}*',
```

Comment:
Check comment on NamingConvention - this prefix restriction only makes sense if we enforce naming convention on Omics Runs right?

Reply:
Yes, I am not sure if the self.env_resource_prefix portion is needed. I think we need the '*' but the tag condition key should give the authorization protection I was looking to enforce.